drv/stm32h7-update-server/src/main.rs

```diff
@@ -521,6 +521,15 @@ impl idl::InOrderUpdateImpl for ServerImpl<'_> {
         }
         Ok(chunk.len() as u32)
     }
+
+    fn switch_default_image(
+        &mut self,
+        _: &userlib::RecvMessage,
+        slot: SlotId,
+        duration: SwitchDuration,
+    ) -> Result<(), RequestError<UpdateError>> {
+        Err(UpdateError::Unknown)
```

Comment:
Do we eventually want to implement this in terms of `swap_banks`?

Reply:
Probably? I'm not actually sure how bank swapping is handled right now.